src/ProjectSystemTools/BuildLogging/Model/Build.cs

```diff
@@ -3,104 +3,42 @@
 using System;
 using System.Collections.Generic;
 using System.IO;
-using System.Linq;
-using Microsoft.VisualStudio.ProjectSystem.Tools.TableControl;
-using Microsoft.VisualStudio.Shell.TableManager;
+using Microsoft.VisualStudio.ProjectSystem.Tools.BuildLogging.Model.Backend;
+using Microsoft.VisualStudio.ProjectSystem.Tools.Providers;
 
 namespace Microsoft.VisualStudio.ProjectSystem.Tools.BuildLogging.Model
 {
+    // server side data (deals with log files)
     internal sealed class Build : IComparable<Build>, IDisposable
     {
-        public BuildType BuildType { get; }
-
-        public IEnumerable<string> Dimensions { get; }
-
-        public IEnumerable<string> Targets { get; }
-
-        public DateTime StartTime { get; }
-
-        public TimeSpan Elapsed { get; private set; }
-
-        public BuildStatus Status { get; private set; }
-
-        public string ProjectPath { get; }
-
+        // TODO: If needed, make an issue: implement more robust ID system (that doesn't use ints)
+        public static int SharedBuildID { get; private set; }
+        public BuildSummary BuildSummary { get; private set; }
```

Comment:
Consider whether `BuildSummary` needs to be made public, or whether the previous properties that were removed above could delegate to an internal summary field instead (eg: `public BuildType BuildType => _summary.BuildType;`).

Using delegation would preserve the API for consumption elsewhere in this PR.